packages/design-system-tailwind-preset/CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.0]
+
+### Added
+
+- Added support for monochromatic button design tokens ([#709](https://github.com/MetaMask/metamask-design-system/pull/709))
+- Added background color tokens: section, subsection, muted ([#682](https://github.com/MetaMask/metamask-design-system/pull/682))
+- Added DisplayLG typography token support ([#607](https://github.com/MetaMask/metamask-design-system/pull/607))
+
+### Changed
+
+- **BREAKING:** Updated peer dependency to `@metamask/design-tokens@^8.0.0` which includes several breaking changes:
+  - Font family tokens now use base font families (`--font-family-default`, `--font-family-accent`, `--font-family-hero`) instead of individual typography variant font families ([#533](https://github.com/MetaMask/metamask-design-system/pull/533))
+  - Primary font changed from Euclid Circular B to CentraNo1 ([#499](https://github.com/MetaMask/metamask-design-system/pull/499))
+  - Typography font size updates for Body variants on small screens ([#533](https://github.com/MetaMask/metamask-design-system/pull/533))
+- Updated typography configuration to use new base font family tokens ([#699](https://github.com/MetaMask/metamask-design-system/pull/699))
+- Updated build pipeline to use ts-bridge ([#704](https://github.com/MetaMask/metamask-design-system/pull/704))
+
```

Comment:
Updating changelog with what's been added and breaking changes from design tokens for the tailwind preset package